app/controllers/batch_edits_controller.rb

```diff
@@ -7,4 +7,30 @@ def form_class
     BatchEditForm
   end
 
+  # nearly straight-up copy of sufia's method
+  def update
```

Comment:
I assume "nearly" straight-up copy of sufia's method means it's not an _exact_ copy... or there'd be no reason to copy it and override it? Would it be good for the comment to say what the "nearly" is?